util/conversion/conversion.go

```diff
@@ -24,7 +24,8 @@ import (
 	"testing"
 
 	fuzz "github.com/google/gofuzz"
-	"github.com/onsi/gomega"
+	. "github.com/onsi/gomega"
```

Comment:
This isn't a test file, so maybe use gomega directly?

Reply:
done